Watchman.Discord/Areas/Statistics/Controllers/StatisticsController.cs

```diff
@@ -51,14 +51,15 @@ public void GetStatisticsPerPeriod(DiscordRequest request, Contexts contexts)
         {
             //TODO it doesn't looks clear...
             var period = _reportsService.SelectPeriod(request.OriginalMessage); //TODO use DiscordRequest properties
-            var messages = this.queryBus.Execute(new GetMessagesQuery()).Messages;
-            var report = _reportsService.CreateReport(messages, period, contexts.Server);
+            var getMessages = new GetMessagesQuery(contexts.Server.Id);
+            var messages = this._queryBus.Execute(getMessages).Messages.ToList();
+            var report = _reportsService.CreateReport(messages, period);
             Log.Information("Generated statistics for time range {start} {end}", report.TimeRange.Start, report.TimeRange.End);
-            var messagesService = messagesServiceFactory.Create(contexts);
+            var messagesService = _messagesServiceFactory.Create(contexts);
 #if DEBUG
             //TODO it should be inside messages service... or responses service
             var dataToMessage = "```json\n" + JsonConvert.SerializeObject(report.StatisticsPerPeriod.Where(x => x.MessagesQuantity > 0), Formatting.Indented) + "\n```";
-            messagesService.SendMessage(dataToMessage);
+            Log.Information(dataToMessage);
```

Comment:
to będzie zawierać też m.in ```json\n
myśle że wystarczy informacja ogólna 
krótkie podsumowanie jednym zdaniem

Reply:
To i tak jest tylko wypisywane do logów i tylko podczas Debug, zostawię na razie tak jak jest. Jak będzie kiedyś przeszkadzać to się zmieni